Update DCP_change_log.md (#2732)
SkylineCommunications/dataminer-docs: release-notes/Cloud/DCP_change_log.md

> [!NOTE]
> Many features on dataminer.services are dependent on DxMs. You can find the change logs for these under [DxM release notes](xref:DxM_RNs_index).

#### 16 February 2024 - Enhancement - Changed user role required to renew system tokens [ID_38722]

> Before the  user needed to be an Owner of the DMS on dataminer.services to be able to [renew the tokens](xref:SLNetClientTest_debugging_cloud_connection). From now on the user only needs the Admin role. Owners can still renew the tokens.

Reviewer: I'd prefer not to link to the SLNetClientTest tool procedure here, since users should really only go there when all other options are exhausted.

```suggestion
Previously, users had to have the role of Owner of the DMS on dataminer.services to be able to [renew the session](xref:Cloud_Connection_Issues#check-the-cloud-session). From now on, this is also possible for users who have the Admin role.
```
